tests/scripts/run-xunit-perf.cmd

```diff
@@ -183,7 +193,12 @@ rem ****************************************************************************
     shift
     goto :parse_command_line_arguments
   )
-
+  IF /I [%1] == [-group] (
```

Comment:
Could you please change [%1] to [%~1]? If user wraps the arguments in double quote, then the condition will fail. The ~ removes the double quotes from the argument before comparing it.